### PR TITLE
{keyvault}Add test for keyvault key get-policy-template

### PR DESCRIPTION
**Pull request by YuanyuanNi**
**Description**

Add test for keyvault key get-policy-template, below test will be affected:
test_keyvault_softdelete

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
keyvault

**Review by evelyn-ys — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py [hunk 1]
```diff
@@ -2206,6 +2206,10 @@ def test_keyvault_softdelete(self, resource_group, key_vault, key_vault2):
         self.cmd('keyvault key create --vault-name {kv} -n key2 -p software',
                  checks=self.check('attributes.enabled', True))
 
+        # test key get-policy-template
+        self.cmd('keyvault key get-policy-template',
+                 checks=self.check('length(@)', 2))
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py:2211]: The `get-policy-template` result is a `json` not a `list`, why check the `length(@)`?

Comment (YuanyuanNi) [reply src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py:2211]: It aims to make sure the command works well and have results return.

Comment (evelyn-ys) [reply src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py:2211]: How about add some meaningful checks... Even checking the result not `None` also makes sense.

Comment (YuanyuanNi) [reply src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py:2211]: This check just means not none.

**Pull request merged**